test/models/client_test.rb

```diff
@@ -26,4 +26,14 @@ def setup
     result = Client.need_accompanying
     assert_equal [], result.to_a
   end
+
+  test 'schedules build correctly automaticly' do
```

Comment:
`automatically` 👮 

(I'm a stickler for correct grammar, feel free to ignore any comments from me with a 👮)